### PR TITLE
feat: add login component

### PR DESCRIPTION
**Pull request by dcocking**
### Type of change

- [ ] Chore
- [X] Feature
- [ ] Bug fix

### Linked issue
Same as this ticket, minus hosting: https://cybrid.atlassian.net/browse/CYB-635

### Why
We need to make the demo application more accessible

### How
Added a login component

Screen Shot 2022-08-12 at 1 09 21 PM

### CODE REVIEW

File: library/src/shared/constants/test.constants.ts [hunk 1]
```diff
@@ -22,7 +22,6 @@ export class TestConstants {
...
 
-  // Demo app customer default for Cypress e2e testing
   static CUSTOMER_GUID = '378c691c1b5ba3b938e17c1726202fe4';
```
Comment (brent-cybrid) [inline library/src/shared/constants/test.constants.ts:25]: This should be an env var.

File: src/demo/components/app/app.component.html [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
+<ng-container *ngIf="demoConfigService.config$ | async as config">
+  <div
+    class="wrapper mat-body"
+    [ngClass]="
+      config.theme === 'LIGHT' ? 'cybrid-light-theme' : 'cybrid-dark-theme'
+    "
+  >
+    <mat-toolbar color="primary">
+      <img
+        src="https://assets-global.website-files.com/6226732e4130814a4adb86c2/62430bcedab2d5494d20b601_logo-white.svg"
```
Comment (brent-cybrid) [inline src/demo/components/app/app.component.html:10]: This should be in a constant.

File: src/demo/components/demo/demo.component.ts [hunk 3]
```diff
@@ -79,36 +67,46 @@ export class DemoComponent implements OnInit, OnDestroy {
...
   }
 
-  initDemo() {
+  initDemo(credentials: DemoCredentials) {
+    this.login$.next(true);
+
+    let config = Constants.DEFAULT_CONFIG;
+    config.customer = credentials.customer;
+    // config.theme = 'DARK';
+    // config.locale = 'fr-CA';
```
Comment (brent-cybrid) [inline src/demo/components/demo/demo.component.ts:91]: Remove

**Review by brent-cybrid — APPROVED** [review]

File: src/demo/components/demo/demo.component.ts (hunk 3 above)
Comment (dcocking) [reply src/demo/components/demo/demo.component.ts:91]: Done.

**Pull request merged**